### DISCUSSION TITLE
Location of redis jobstore

### DISCUSSION THREAD

**Discussion by sbalasa** [agronholm/apscheduler#552] [Q&A] [answered]
Hi,
Where is this redis jobstore example ?

https://github.com/agronholm/apscheduler/tree/master/examples/jobstores/redis_.py

**Comment by agronholm** [comment, marked as answer]
[This](https://github.com/agronholm/apscheduler/blob/3.x/examples/jobstores/redis_.py)?

**Reply by agronholm** [reply]
Was this not a sufficient answer? You haven't commented or marked this as the answer.

**Reply by sbalasa** [reply]
Thanks
